example/src/components/Example.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, Ref } from "react";
+
+import { Showtime, useShowtime } from "../../../";
+
+function App() {
```

Comment:
Is there an easy was for us to access this for our own manual testing purposes but not show it as an official example as a little TypeScript easter egg? I think that would be ideal if we could, though not required. It's a little weird as is because I can't tell if the example actually works, only that it type checks.

Reply:
> Perhaps we could get what you're suggesting just by moving our Example.tsx to the src dir since it's not really a public example yet?
Yeah, that's probably good for now.

Reply:
Updated. Go ahead and resolve this thread if sufficient.